lib/stripe_mock/request_handlers/helpers/subscription_helpers.rb

```diff
@@ -22,7 +32,15 @@ def custom_subscription_params(plans, cus, options = {})
         start_time = options[:current_period_start] || now
         params = { customer: cus[:id], current_period_start: start_time, created: created_time }
         params.merge!({ :plan => (plans.size == 1 ? plans.first : nil) })
-        params.merge! options.select {|k,v| k =~ /application_fee_percent|quantity|metadata|tax_percent/}
+        keys_to_merge = /application_fee_percent|quantity|metadata|tax_percent|billing|days_until_due/
+        params.merge! options.select {|k,v| k =~ keys_to_merge}
+
+        if options[:cancel_at_period_end] == true
```

Comment:
Ugh, I guess you cherry-picked code so you don't have anything to with this

Reply:
😦